colibre/auto_plotter/metallicity.yml

```diff
@@ -498,6 +498,299 @@ stellar_mass_gas_sf_metallicity_lom_allgals_50:
     - filename: GalaxyStellarMassGasMetallicity/FIREbox.hdf5
     - filename: GalaxyStellarMassGasMetallicity/Strom2022_OH.hdf5
 
+stellar_mass_gas_nitrogen_over_oxygen_lom_50_kpc:
+  type: "scatter"
+  legend_loc: "upper left"
+  selection_mask: "derived_quantities.has_cold_dense_gas_50_kpc"
+  comment: "All galaxies, LoM, Diffuse"
```

Comment:
Should this be updated to reflect it's only galaxies with cold dense gas?

Reply:
I updated all comments to describe galaxies with cold dense gas. Also, in metallicity.ylml I modified the captions of some figures from the stellar metallicity section that needed improving.